src/server/service/kv.rs

```diff
@@ -53,8 +55,11 @@ pub struct Service<T: RaftStoreRouter + 'static> {
     // For handling snapshot.
     snap_scheduler: Scheduler<SnapTask>,
     recursion_limit: u32,
+    batch_row_limit: usize,
+    stream_batch_row_limit: usize,
     metrics: Metrics,
     stream_channel_size: usize,
+    max_handle_duration: Duration,
```

Comment:
The name is not suitable here, at least need the prefix `cop_`.  And I do not think it's a good idea to move the coprocessor's global config `batch_row_limit`,`stream_batch_row_limit `, `request_max_handle_duration ` from `Endpoint::Host` to `Service`.

Reply:
It is becuase we cannot access variables in the `Host` when running coprocessor tasks.

Reply:
So, instead of this, now we access the variable when building the task and the task always carries these variables. In `Service` we build the tasks, thus we access these variables in `Service`.

Reply:
I think these configurations belong to the `Endpoint::Host`, let's put them there.
If you want to make the request context carry these variables, you can use two types, one for sending the task to the `Endpoint::Host`, another for wrapping all you need in the `Endpoint::Host`.